docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:alpine3.13 as jekyll
+
+RUN apk add --no-cache build-base gcc bash cmake git
+
+RUN gem install bundler -v "~>2.2.11" && gem install jekyll
```

Comment:
You shouldn't need to `gem install jekyll` (which installs it globally and doesn't use a specific version), it should come in by running `bundle install` (which installs the version declared in Gemfile or locked by Gemfile.lock).

Reply:
Agree and thanks for the catch. Will remove it from it.